definitions/variable/emissions/tag_carbon-capture-source.yaml

```diff
@@ -0,0 +1,9 @@
+- Carbon Capture Source:
+    - Biomass:
+        description: purpose-grown bioenergy crops, biogas, crop and forestry residue,
+          municipal solid waste bioenergy and traditional biomass
+    - Fossil:
+        description: fossil fuels including coal, natural gas and conventional/unconventional oil
+    - Synthetic Fuels:
+        description: synthetic fuels produced from carbon-capture-and-utilization (CCU),
+          not derived from fossil fuels
```

Comment:
We are missing Industrial Processes, Direct Air Capture and Other.

Reply:
Added directly as variables in `carbon-management.yaml`

Reply:
> Added directly as variables in `carbon-management.yaml`

Definition of synthetic fuels "synthetic fuels produced from carbon-capture-and-utilization (CCU), not derived from fossil fuels" does not work in my view. Synthetic fuels can be produced with CO2 from fossils as well as with CO2 from cement plants.  

Carbon capture|Synthetic Fuel  "Carbon captured in synthetic fuels"
Carbon capture|Synthetic Fuel|Carbon-neutral  "Carbon captured in synthetic fuels produced with carbon captured directly or indirectly (e.g. biogenic carbon) from the air "
Carbon capture|Synthetic Fuel|Fossil   "carbon captured in synthetic fuels produced with carbon captured in fossil fuel combustion processes"
Carbon capture|Synthetic Fuel|Industrial Processes   "synthetic fuels produced with carbon captured in industrial processes (e.g. cement production; excluding fossil fuel combustion)"

Reply:
Agree with Elmar here. We have the choice of either having synthetic fuels as a separate category, or trying to allocate the synthetic fuels to the other sources, depending on the source of the carbon that was used. The latter can get very messy, especially if a model considers a circular economy, where the carbon is captured and used several times. That would lead to an infinite circle of use and re-use, and the only way I see for a clean accounting is to consider synthetic fuels as carbon-neutral and as their own category. Subcategories of synthetic fuels can in principle be done, but I would suggest to omit them as one of the subcategories would have to be synthetic fuels again...

Reply:
Good point, Jessica. Since Synthetic Fuels can be both source as well as destination, they would show up both in Carbon Capture|<source> as well as Carbon Capture|Utilization|<type of use = synthetic fuel> and Carbon Capture|Geological Formation|<type of captured carbon = synthetic fuel>. 

However, I do not think we can restrict all of this to "carbon neutral". The problem of re-cycling is not solved by this and we are missing a significant part of synfuels that are not carbon neutral. Hence I suggest to stick with the subcategorization but be clearer in the definitions.

 "Carbon captured in synthetic fuels produced with carbon captured directly or indirectly (e.g. biogenic carbon or carbon from carbon-neutral synfuels) from the air"
Carbon capture|Synthetic Fuel|Fossil "carbon captured in synthetic fuels produced with carbon captured in fossil fuel combustion processes or from fossil-based synfuels"
Carbon capture|Synthetic Fuel|Industrial Processes    "carbon captured in synthetic fuels originally produced with carbon captured in industrial processes (e.g. carbon captured from cement production or from synfuels derived with carbon from industrial processes). Fossil-fuel based synthetic fuels are excluding here."

Of course, if synthetic fuels from different origins are mixed at some point, and then the carbon is captured from this mix, the separation becomes harder (would need to resort to fractions of the mix). But the same is true if we want to restrict just to "carbon neutral synfuels".

Reply:
I was only suggesting to treat the synfuels as carbon neutral, and report the emissions where the CO2 is captured. But that is another discussion. We will most likely not be able to report the subcategories in REMIND, but there is also no harm in having the variables here. So agree with Elmars suggestion. After that we should be good to merge!